### PR TITLE
Create pull requests to bump an analyzer via GitHub Actions

### LINKED ISSUE #130: Make update PR automtically
For tools without Dependabot update support (e.g. SwiftLint), I'd like to introduce some auto-update PR system.

Perhaps I think that writing own GitHub Actions can realize this.

### PR DESCRIPTION
**Pull request by ybiquitous**
## Summary

- Add a new Rake task `bump_analyzers`.
- Run the task on schedule.

## See also

- https://help.github.com/en/github/automating-your-workflow-with-github-actions/virtual-environments-for-github-actions#default-environment-variables
- https://help.github.com/en/github/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions

Fixes #130

### CODE REVIEW

**Comment by ybiquitous** [comment]
Generated PRs: #351 #352

File: .github/workflows/bump_analyzers.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Bump analyzers
+
+on:
+  schedule:
+    - cron: "0 0 */1 * *"
```
Comment (ybiquitous) [inline .github/workflows/bump_analyzers.yml]: [note] We should verify the setting after this PR will be merged. If succeeded, we should increase interval days like `0 0 */30 * *`.

File: lib/tasks/bump_analyzers.rb [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  end
+
+  # @see https://developer.github.com/v3/git/refs/#list-matching-references
+  def tags
+    @tags ||= call_github_api(:get, "/repos/#{github_repo}/git/refs/tags")
+      .map { |tag| tag.fetch(:ref).delete_prefix("refs/tags/") }
+  end
+
+  def latest_version
+    @latest_version ||= tags.last.delete_prefix("v")
```
Comment (yykamei) [inline lib/tasks/bump_analyzers.rb]: `tags.last` is really the latest version? I think we should consider pagination when calling GitHub APIs.

Comment (ybiquitous) [reply lib/tasks/bump_analyzers.rb]: Yes, ideally. But I think the pagination code will be too complex here. Do you have any solutions?

Comment (yykamei) [reply lib/tasks/bump_analyzers.rb]: Probably, the only solution is to use GitHub API v4... yes, I know it's also complex.

https://developer.github.com/v4/object/repository/

Screenshot from 2019-10-24 11-14-37

Comment (ybiquitous) [reply lib/tasks/bump_analyzers.rb]: Umm..., the current code seems to work enough, so how about considering after any problems will occur?
(It seems to me that Octokit.rb is easier than the raw GraphQL API 😅 )

Comment (yykamei) [reply lib/tasks/bump_analyzers.rb]: I thought all GitHub APIs have pagination, but I tried the same endpoint `refs` on my local machine, then GitHub responds without pagination information. It looks pagination does not affect this endpoint.

OK. It works well. Sorry for preventing you.

**Comment by ybiquitous** [comment]
Thank you for reviewing! 😄

**Pull request merged**